source/administration/incident-response-application.rst

```diff
@@ -0,0 +1,667 @@
+Incident Response Application (EE, Closed Alpha)
+------------------------------------------------------
+
+The incident response application is available in closed Alpha, in `Enterprise Edition <https://mattermost.com/pricing/>`_. Supported in Mattermost 5.12 and later.
+
+Use the incident response application to connect all your workflows, automate repetitive tasks and collaborate on incidents within one secure messaging platform. Sample use cases you can accomplish with this app include the following:
+
+1. Trigger automated incident response workflows based on keywords.
+2. Automatically mention your InfoSec or DevSecOps teams when an incident occurs, including via email, mobile push and desktop notifications.
+3. Auto-create "war rooms" and invite key team members to immediately collaborate on a critical incident.
+4. Take quick actions, review data and access relevant links all in one place.
+5. Archive resolved incidents to declutter your channel sidebar without losing access to past information.
+6. Pull summary statistics of the incident response workflow, including mean-time-to-acknowledgment (MTTA) and mean-time-to-resolution (MTTR).
+
+Below is an example of the incident response app automatically creating a channel from a Nagios alert, adding users to the channel and providing them quick actions to take on the incident.
+
+.. image:: ../images/incident-response-app-intro-image.png
+   :alt: Incident Response App: Intro Image
+
+.. image:: ../images/incident-response-app-intro-image-fullpage.png
```

Comment:
@amyblais Question: Do images typically have a border? If yes, how is it added (is it locally via image editing program)?

Reply:
This should be a question for UX. @asaadmahmood